src/scss/_footer.scss

```diff
@@ -104,30 +104,34 @@ footer {
 }
 
 @media #{$phone-media-query} {
+  footer {
+    padding-top: 60px;
```

Comment:
I'm not sure that's the optimal solution. The real problem is that the thank you is over the footer and is hidden. Have you tried playing with ```z-index``` on the footer?